Why: emscripten declares getrandom() but does not implement it; use getentropy() in the FIPS-mode branch too.

diff --git a/random/rndgetentropy.c b/random/rndgetentropy.c
index a6f3c4ab17b60f90b6612a8fe03fa578754a1c98..8bdde1012b373205166bbee26814b340a49b5abc 100644
--- a/random/rndgetentropy.c
+++ b/random/rndgetentropy.c
@@ -96,7 +96,11 @@ _gcry_rndgetentropy_gather_random (void (*add)(const void*, size_t,
                * RHEL kernel.
                */
               nbytes = length < 32 ? length : 32;
+#ifdef __EMSCRIPTEN__
+              ret = getentropy (buffer, nbytes);
+#else
               ret = getrandom (buffer, nbytes, GRND_RANDOM);
+#endif
             }
           else
 #endif
